xbmc/application/Application.h

```diff
@@ -218,6 +219,7 @@ class CApplication : public IWindowManagerCallback,
 
   std::chrono::time_point<std::chrono::steady_clock> m_lastRenderTime;
   bool m_skipGuiRender = false;
+  std::optional<bool> m_guiRenderLastState;
```

Comment:
is std::optional really needed or would just a bool also do the job?

Reply:
looks like it's used to avoid marking window dirty in initial render, but idk how required it is